source/Calamari.Common/Plumbing/Extensions/AesEncryption.cs

```diff
@@ -8,6 +8,21 @@ namespace Calamari.Common.Plumbing.Extensions
 {
     public class AesEncryption
     {
+        //Key size used to encrypt variables for scripts (bash/powershell etc.)
+        //The variables are decrypted in the respective bootstrapper scripts
+        public static readonly int ScriptBootstrapKeySize = 256;
+        
+        //Key size used to decrypt the variables file sent by Octopus Server
+        public static readonly int VariablesFileKeySize = 128;
```

Comment:
SaST will update this once they sort out variable encryption on the Server side. Currently it's encrypted by Tentacle which sounds wrong